game/glutil.cc

```diff
@@ -24,6 +24,8 @@ namespace glutil {
 
 	void VertexArray::clear() {
 		m_vertices.clear();
+		glDeleteVertexArrays(1, &m_vao);
+		glDeleteBuffers(1, &m_vbo);
```

Comment:
Here's a link to support why I don't delete them here:

https://community.khronos.org/t/understanding-vaos-vbos-and-drawing-two-objects/72778/2